CHANGELOG.log

```diff
@@ -1,3 +1,7 @@
+June 25th, 2015
```

Comment:
Conversation on IRC : we should only update the version after the merge. Multiple merge could be done on the master branch and then we should update the version number.